core/drivers/crypto/caam/include/caam_desc_defines.h

```diff
@@ -420,6 +423,9 @@
  */
 #define CMD_JUMP_TYPE		CMD_TYPE(0x14)
 
+/* Jump Select Type */
+#define JMP_JSL BIT32(24)
```

Comment:
maybe a tab, for consistency with other macros around.

Reply:
Usually, clang-format only uses tabs to align successive macros. But for consistency in this file, I will use tabs.